packages/devtools_app/lib/src/screens/memory/memory_controller_discoverable.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/foundation.dart';
+
+import '../../../devtools_app.dart';
+import '../../analytics/constants.dart' as analytics_constants;
+import '../../extensibility/discoverable.dart';
+import 'memory_tabs.dart';
+
+class DiscoverableMemoryPage extends DiscoverablePage {
+  DiscoverableMemoryPage(this.controller) : super() {
+    discoverableApp.memoryPage = this;
+  }
+
+  final MemoryController controller;
+
+  static String get id => MemoryScreen.id;
+
+  // Events
+  static const memorySnapshotTaken = 'mem-snapshot-done';
```

Comment:
could be nice to have a single enum of all the events so it is easier to track them across all screens.
The enum could have comments about what page they are for.